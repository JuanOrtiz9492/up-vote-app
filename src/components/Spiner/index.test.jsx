import React from 'react';
import { render } from '@testing-library/react';
import Spiner from '.';

describe('Spiner Component', () => {

    it('should Match the Snapshot', () => {
        const { container } = render(<Spiner />);
        expect(container).toMatchSnapshot();
      })
})

