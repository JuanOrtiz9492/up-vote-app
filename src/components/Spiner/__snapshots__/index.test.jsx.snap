// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spiner Component should Match the Snapshot 1`] = `
<div>
  <div
    class="sc-AykKC ejRNwF"
  />
</div>
`;
